frameworks/helloworld/src/main/dist/examples/overlay.yml

```diff
@@ -1,6 +1,7 @@
 name: {{FRAMEWORK_NAME}}
 scheduler:
   user: {{SERVICE_USER}}
+  principal: {{SERVICE_PRINCIPAL}}
```

Comment:
`SERVICE_PRINCIPAL` was already defined in config.json and marathon.json and just not used before?